scripts/Install-NetFramework.ps1

```diff
@@ -0,0 +1,55 @@
+﻿[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]
+    $DomainNetBIOSName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $DomainAdminUser,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $DomainAdminPassword
+)
+
+
+try {
+    Start-Transcript -Path C:\cfn\log\Install-NetFramework.ps1.txt -Append
+    $ErrorActionPreference = "Stop"	
+
+    $DomainAdminFullUser = $DomainNetBIOSName + '\' + $DomainAdminUser
+    $DomainAdminSecurePassword = ConvertTo-SecureString $DomainAdminPassword -AsPlainText -Force
+    $DomainAdminCreds = New-Object System.Management.Automation.PSCredential($DomainAdminFullUser, $DomainAdminSecurePassword)
+
+    $InstallNetfwPs={
+        $ErrorActionPreference = "Stop"
+        $InstallPath = "C:\Exchangeinstall\NDP462-KB3151800-x86-x64-AllOS-ENU.exe"
```

Comment:
So if 4.6.2 is in the latest AMIs, then we surely don't need to install it with this script.